firmware/lib/bip32/bip32.c

```diff
@@ -13,11 +13,12 @@
 #define KEY_IDENTIFIER_SIZE (HASH160_DIGEST_SIZE)
 
 static bool all_zeroes(uint8_t* buf, uint32_t size) {
```

Comment:
Unrelated to your change, but we can(and should) can also make buf a const *

Reply:
Good idea. I'll add a commit.

Reply:
Shouldn't the `size` parameter also be a `size_t`?

Reply:
Thanks both! We should have made this more clear, but this repo is actually a mirror to an internal repo which is ahead; we'll publish squashed commits regularly.

I will take these changes and add them to the internal repo, and you'll see them reflected here, but it won't be via this PR.